umap/static/umap/js/umap.js

```diff
@@ -1505,8 +1505,15 @@ U.Map = L.Map.extend({
       L.DomUtil.createButton(
         'button umap-empty',
         advancedButtons,
-        L._('Empty'),
-        this.empty,
+        L._('Clear data'),
+        this.emptyDatalayers,
+        this
+      )
+      L.DomUtil.createButton(
+        'button umap-empty',
+        advancedButtons,
+        L._('Remove layers'),
+        this.emptyMap,
```

Comment:
Maybe align and name it "removeLayer"? (or maybe it's already taken, and we should suffix these with "Action" or something similar ?)